packages/sdk/src/conversation/middleware.ts

```diff
@@ -120,33 +132,38 @@ export class CommandResponseMiddleware implements Middleware {
   }
 
   private classifyActivity(activity: Activity): ActivityType {
-    if (this.isCommandReceived(activity)) {
-      return ActivityType.CommandReceived;
+    if (activity.type === ActivityTypes.Message) {
+      return ActivityType.CurrentBotMessaged;
     }
 
     return ActivityType.Unknown;
   }
 
-  private isCommandReceived(activity: Activity): boolean {
-    if (this.commandHandlers) {
-      const commandText = this.getActivityText(activity);
-      const handlers = this.filterCommandHandler(commandText, this.commandHandlers);
-      return handlers.length > 0;
-    } else {
-      return false;
+  private matchPattern(pattern: string | RegExp, text: string): boolean | RegExpMatchArray {
+    if (text) {
+      if (typeof pattern === "string") {
+        const regExp = new RegExp(pattern as string, "i");
+        return regExp.test(text);
+      }
+
+      if (pattern instanceof RegExp) {
+        const matches = text.match(pattern as RegExp);
+        return matches ?? true;
```

Comment:
`matches ?? false`? Null `matches` means not matched.

Reply:
Yep. Should be a bug and already fixed.